src/compiler/performanceCore.ts

```diff
@@ -73,8 +77,8 @@ namespace ts {
             try {
                 let performance: Performance;
                 const { performance: nodePerformance, PerformanceObserver } = require("perf_hooks") as typeof import("perf_hooks");
-                if (hasRequiredAPI(nodePerformance, PerformanceObserver)) {
-                    performance = nodePerformance;
+                if (hasRequiredAPI(nodePerformance as unknown as Performance, PerformanceObserver)) {

```

Comment:
Is this because our reference to `@types/node` is out of date? `clearMeasures` as added in [NodeJS v16.7.0](https://nodejs.org/dist/latest-v18.x/docs/api/perf_hooks.html#performanceclearmeasuresname).

Reply:
Yes its because of our `@types/node`

Reply:
Any reason we shouldn't update our local copy of `@types/node` to at least v16?